packages/2018-disaster-resilience/src/components/SignificantStructuralDamage/index.js

```diff
@@ -13,7 +29,33 @@ export class SignificantStructuralDamage extends React.Component {
         title="Significant Structural Damage"
         slug="significant-structural-damage"
       >
-        <Placeholder />
+        <Collapsable>
+          <Collapsable.Section>
+            <div>
+              <p>Key pieces of Portland&#39;s infrastructure are estimated to be destroyed or damaged beyond a usable state in a 9.0 Cascadia Subduction Zone earthquake. Three specific weaknesses will severely inhibit the ability to rescue people, distribute services, and rebuild after a Cascadia quake.</p>
+              <ChartContainer title="Critical Infrastructure Rendered Unusable">
+                <div className={infographicContainer}>
+                  <div className={infographicItem}>
+                    <img src="https://s3-us-west-2.amazonaws.com/hacko-cdn/2018-disaster-resilience/CriticalEnergy.svg" className={infographicImg} />
```

Comment:
Nice bucket name. Now we just need to slap some DNS on it. Maybe `cdn.civicplatform.org` or `static.civicplatform.org`.